packages/lesswrong/components/comments/CommentsList.jsx

```diff
@@ -52,6 +52,7 @@ class CommentsList extends Component {
           <div className="comments-list">
             {comments.map(comment =>
               <Components.CommentsNode
+                totalCommentCount={comments.length}
```

Comment:
I believe this comment count is the number of comments at a single level, since `comments` here is the root of a treelike structure. So if there are 5 comments with 10 replies each, that would only count as 5, and wouldn't cross the 25-comment collapsing threshold, somewhat defeating the purpose. Instead I think we want a function which walks the tree and counts the number of comments recursively.

Reply:
Ah, it turned out this count was stored a couple levels up the component tree already, and is now passed down.